### PR TITLE
Implement two APIs for POI resources, `add` and `get`.

### PR DESCRIPTION
**Pull request by lemonlatte**
There are two new APIs introduced:
1. `add`: this is to add a resource into a POI so it could be rated later on
2. `get`: this API is to get suggest resources from the official list. It will exclude items that have already added into a POI.

Changes Highlight:
- In order to cache the official resources, we implement a new function `LoadDefaultPOIResources`. This should be loaded before APIs is called.
- Add two translation file for resources.
- Add `omitempty` for `Resources` of `POIRatingsMetric`. If we do not set this, the resource would be set to `resources: null` in mongodb. And break the `$push` command.
- Add `PanicIfError` into utils package

### CODE REVIEW

File: schema/poi.go [hunk 1]
```diff
@@ -37,3 +38,17 @@ type POIDetail struct {
 	ProfilePOI `bson:",inline"`
 	Location   *Location `json:"location"`
 }
+
+type ResourceSource string
+
+const (
+	OfficialResource   ResourceSource = "official"
+	CustomizedResource ResourceSource = "customized"
```
Comment (pieceofr) [inline schema/poi.go:46]: Is this CustomizedResource is for future?

File: schema/poi.go [hunk 2]
```diff
@@ -37,3 +38,17 @@ type POIDetail struct {
...
+)
+
+type Resource struct {
+	ID     string         `json:"id" bson:"_id"`
+	Name   string         `json:"name" bson:"name"`
+	Desc   string         `json:"-" bson:"desc"`
```
Comment (pieceofr) [inline schema/poi.go:52]: I don't think the list including Desc and Type. Can we omit it ?

File: store/poi_test.go [hunk 3]
```diff
@@ -561,6 +624,119 @@ func (s *POITestSuite) TestUpdatePOIAliasForNotAddedPOI() {
...
+	s.NoError(err)
+	s.Len(resources, 30)
+}
+
+func (s *POITestSuite) TestGetPOIResourcesWithNoDefaultLanguage() {
+	delete(defaultResourceIDMap, "en")
+	delete(defaultResourceList, "en")
+
+	store := NewMongoStore(s.mongoClient, s.testDBName)
+	resources, err := store.GetPOIResources(noResourcesPOIID2, false, "fr")
```
Comment (nodestory) [inline store/poi_test.go:720]:
```suggestion
	resources, err := store.GetPOIResources(noResourcesPOIID2, false, "en")
```

Comment (lemonlatte) [reply store/poi_test.go]: Fixed

**Pull request merged**